appservice/src/tree/AppSettingTreeItem.ts

```diff
@@ -77,4 +77,28 @@ export class AppSettingTreeItem extends AzureTreeItem<ISiteTreeRoot> {
         this._hideValue = !this._hideValue;
         await this.refresh();
     }
+
+    public async toggleSlotSetting(): Promise<void> {
+        const slotSettings: SlotConfigNamesResource = await this.root.client.listSlotConfigurationNames();
+        if (slotSettings.appSettingNames) {
+            const slotSettingIndex: number = slotSettings.appSettingNames.findIndex((value: string) => { return value === this._key; });
```

Comment:
How does case sensitivity work for this? Can I have two settings with different casing?

Reply:
The settings are case sensitive across the slots (including production).  So SETTING and setting are considered to be different slot settings.  However, settings aren't case sensitive within one slot, so you wouldn't be able to have setting and SETTING in the same slot.